src/main/bindings/rust/CMakeLists.txt

```diff
@@ -34,18 +35,12 @@ add_custom_command(OUTPUT wrapper.rs
         --whitelist-function "syscallcondition_new"
         --whitelist-function "syscallcondition_unref"
         --whitelist-function "syscallhandler_.*"
-        --whitelist-function "worker_getActiveProcess"
-        --whitelist-function "worker_getActiveThread"
-        --whitelist-function "worker_readPtr"
-        --whitelist-function "worker_writePtr"
-        --whitelist-function "worker_getReadablePtr"
-        --whitelist-function "worker_getWritablePtr"
-        --whitelist-function "worker_getMutablePtr"
-        --whitelist-function "worker_flushPtrs"
+        --whitelist-function "worker_.*"
```

Comment:
I'm just worried that if we need to support distros that have different versions of GLib, then it might be difficult to provide bindings that are valid for both versions. I agree with the comments in your commit message, there are other ways we could work around this, but I think we should also try to avoid having to ever work with GLib types from Rust.